File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
  int height;
  do
  {
    height = get_int("Height: ");
  }
  while(height < 1 || height > 8);

  for(int i = 1; i <= height; i++)              // for adding coloumns
    {
      for(int n = height - 1 ; n >= i ; n--)    //for printing empty spaces in descending pyramid
      {
        printf(" ");
      }
      for(int j = 1 ; j <= i ; j++)             // for printing hash rows to form ascending pyramid
      {
        printf("#");
      }
        printf("  ");
      for(int j = 1 ; j <= i ; j++)
      {
        printf("#");
      }
      printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int
# Get height from user
while True:
    h = get_int("Pyramid height = ")
    if h <= 8 and h >= 1:
        break

i = 0
while (i <= h-1):
    # print right aliged pyramid
    print(" " * (h-i-1) + "#" * (i+1), end="")
    # print spaces between pyramids
    print("  ", end="")
    # print left aligned pyramid
    print("#" * (i+1))
    i += 1